### ISSUE TITLE
Settings button icon in Events has insufficient contrast against its background in 'selected' state

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#879]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![26 1](https://user-images.githubusercontent.com/22068434/88357551-85766d80-cd20-11ea-983c-7bdaa21170ad.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1384.01. Please validate/close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1388.01. All 6 modes are now compliant
